### ISSUE TITLE
Issues with links in Markdown editor with plaintext Content API

### ISSUE THREAD

**Issue by dandesz198** [TryGhost/Ghost#11023]
### Issue Summary

When using the "Markdown section" in the editor to add a link, for example, `[Example](http://example.com)`, everything seems to be working fine.
When requesting the `html` field for the page via the GraphQL API, I get the `<a>` tag correctly; however, when requesting the `plaintext` format so that I can parse Markdown myself, I get a wrongly formatted Markdown link:
```
Example [http://example.com]
```
The issue only seems to be happening when I use the Markdown editor inside the editor. If I start writing a Markdown link somewhere inside the page and outside a Markdown field, it works properly.

(I use the Markdown editor for the .md syntax highlighting).

The Ghost itself is self-hosted and runs inside a Docker container. I used the official `ghost` image.

### To Reproduce

1. Deploy a Ghost inside a Docker container somewhere
2. Create a new page inside Ghost
3. Add a link inside the page with the `[]()` syntax in the general editor, and add a Markdown section and add a link inside that, too.
4. Request the page's `HTML` and `plaintext` properties via the Content API, then log them to the console or something

### Technical details:

* Ghost Version: 2.27.0
* Browser/OS: editing from a Chrome 76.0.3809.100
* Database: sqlite3

**Comment by kevinansfield** [comment]
> when requesting the plaintext format so that I can parse Markdown myself

I think you are misunderstanding what the `plaintext` format is. It's not markdown, it's the post contents converted into a human-readable plain text format format. The raw format is stored as mobiledoc which is not available via the Content API.

If you need help or support for your use-case, please pop by [our forum](https://forum.ghost.org) - we use GitHub solely for bug tracking and on-going feature work 🙂

**Comment by AndrewRayCode** [comment]
FWIW, I found this issue when searching for ghost blog content api excerpt links. My issue is that when using the content API to get the excerpt, and rendering that into my site, long links inside of brackets, which at first glance look like unescaped markdown, are part of the excerpt. For long links, this makes the unbroken text overflow the center column. I didn't realize this was intentional behavior of the excerpt.

For anyone else who finds this ticket, I solved it with:

```javascript
const process = (html: string) => html.replace(/\[[^\]]+\]\s*/g, '');
```

and

```jsx
<p>{process(post.excerpt)}&hellip;</p>
```
